include/dicom/dicom.h

```diff
@@ -858,7 +858,7 @@ bool dcm_element_set_value_sequence(DcmError **error,
  * :param indentation: Number of white spaces before text
  */
 DCM_EXTERN
-void dcm_element_print(const DcmElement *element, uint8_t indentation);
+void dcm_element_print(const DcmElement *element, int indentation);
```

Comment:
We wanted to use exact-width integer types in the public API. What is the motivation for switching to `int`?

Reply:
Sized int types are useful when variables are derived from a file format with specific numeric size. For example, DICOM elements have a length in a four byte unsigned value, so `uint32_t` is meaningful as the return value for  `dcm_element_get_length()`.

`indentation` however is an abstract value and does not relate to any concrete representation. Giving it specific format adds complexity for no benefit, and I think the extra complexity actually makes bugs more likely.